submissions/views.py

```diff
@@ -185,18 +202,22 @@ def panel(request):
 
 @xframe_options_exempt
 @csrf_exempt
-def panelist(request):
+def panelist(request, conslug="ConFusion2020"):
```

Comment:
Is `"Confusion2020"` supposed to be hard-coded here or should it just be `conslug`?